data/faq20.toml

```diff
@@ -32,6 +35,39 @@ q = "Is short polling intervals a problem?"
 a = "Polling interval really only impacts the time-to-activation (scaling from 0 to 1) but once scaled to one it's really up to the HPA (horizontal pod autoscaler) which polls KEDA."
 type = "Features"
 
+[[qna]]
+q = "Using multiple triggers for the same scale target"
+a = """
+KEDA allows you to use multiple triggers as part of the same `ScaledObject` or `ScaledJob`.
+
+By doing this, your autoscaling becomes better:
+- All your autoscaling rules are in one place
+- You will not have multiple `ScaledObject`'s or `ScaledJob`'s interferring with each other
+
+KEDA will start scaling as soon as when one of the triggers meets the criteria.
```

Comment:
I'd add a note that the maximum number is used for the target replica count calculation

Reply:
You mean the maximum number of what? The configured value for that trigger or?

Reply:
I meant max number acrross multiple triggers. Let' say you have mutliple triggers and each reports different metrics eg: trigger1: 5, trigger2: 10, trigger3: 0. Then HPA will  do the calculation on each of these metrics and will select the largest number for desired replica count.

Reply:
What about this?
```suggestion
KEDA will start scaling as soon as when one of the triggers meets the criteria. Horizontal Pod Autoscaler (HPA) will calculate metrics for every scaler and use the highest desired replica count to scale the workload to.
```

Reply:
Perfect :)